server/helpers/ffmpeg-utils.ts

```diff
@@ -13,6 +13,17 @@ function getVideoFileHeight (path: string) {
   })
 }
 
+function getVideoFileStreams (path: string) {
+  return new Promise<any>((res, rej) => {
+    ffmpeg.ffprobe(path, (err, metadata) => {
+      if (err) return rej(err)
+
+      const videoStream = metadata.streams.find(s => s.codec_type === 'video')
```

Comment:
Throw an error if `metadata.streams` does not exists or if we can't find a video stream. This will be very useful for logging :)